closure/closure-compiler/src/com/google/javascript/jscomp/JsMessage.java

```diff
@@ -164,7 +164,7 @@ public String getDesc() {
    * Gets the meaning annotated to the message, intended to force different
    * translations.
    */
-  String getMeaning() {
+  public String getMeaning() {
```

Comment:
Back in the day, Michael carried local patches to the closure things. Has
that practice been discontinued?
On Mar 16, 2016 8:51 AM, "Nick Santos" <notifications@github.com> wrote:

> In
> closure/closure-compiler/src/com/google/javascript/jscomp/JsMessage.java
> <https://github.com/bolinfest/plovr/pull/95#discussion_r56359138>:
>
> > @@ -164,7 +164,7 @@ public String getDesc() {
> >     * Gets the meaning annotated to the message, intended to force different
> >     * translations.
> >     */
> > -  String getMeaning() {
> > +  public String getMeaning() {
>
> it seems like this is something that you'd need to contribute upstream?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/bolinfest/plovr/pull/95/files/dc127ed0b26949d173fe76477d677c87d64f2ee4#r56359138>
>